frontend/components/tree/js/tree.service.ts

```diff
@@ -1041,6 +1043,11 @@ export class TreeService {
 	 * Unselect all selected items and clear the array
 	 */
 	public clearCurrentlySelected() {
+
+		if (this.DocsService.state.show) {
```

Comment:
Create a function to disable doc component showing instead of manipulating the variable directly

Reply:
Good point